### PR TITLE
install using cosmos server

### PR DESCRIPTION
**Pull request by tamarrow**
This works with the helloworld package only right now.

### CODE REVIEW

File: dcos/package.py [hunk 1]
```diff
@@ -1636,3 +1607,69 @@ def get_apps_for_framework(framework_name, client):
...
+        :param revision: the package revision to install
+        :type revision: str
+        :param options: package parameters
+        :type options: dict
+        :param app_id: app ID for installation of this package
+        :type app_id: str
+        :rtype: None
+        """
+        url = urllib.parse.urljoin(self.cosmos_url, 'v1/package/install')
+        data = '{"name": "helloworld"}'
```
Comment (tamarrow) [inline dcos/package.py]: I ended up hardcoding this for now. I think there's some issue with parsing out the data, kept getting `400`s using json or other formats

File: dcos/package.py [hunk 2]
```diff
@@ -1636,3 +1607,69 @@ def get_apps_for_framework(framework_name, client):
...
+
+        # Send the descriptor to init
+        init_client.add_app(init_desc)
+
+class Cosmos(PackageManager):
+    """Implementation of Package Manager using Cosmos"""
+    def __init__(self, cosmos_url):
+        self.cosmos_url = cosmos_url
+
+    def install_app(self, pkg, revision, options, app_id):
```
Comment (tamarrow) [inline dcos/package.py]: options isn't used right now, but we'll need it to pass options the user supplies from the command line

File: dcos/package.py (hunk 1 above)
Comment (jsancio) [inline dcos/package.py]: Why hardcode to "helloworld"?

Comment (tamarrow) [inline dcos/package.py]: that's the only package that works right now

**Comment by cruhland** [comment]
LGTM

**Pull request merged**